.gitignore

```diff
@@ -13,6 +13,7 @@ app/build/*
 *.perspectivev3
 *.pbxuser
 xcuserdata
+xcshareddata/
```

Comment:
done

Reply:
Yes IDEWorkspaceChecks.plist is recreated by Xcode, but since it's small and (should be) static for all users, so let's go ahead and add IDEWorkspaceChecks.plist to the repo (not to the .gitignore file) so that it doesn't have to be recreated by everyone.
Since xcshareddata is shared, we don't want to blanket-ignore the whole folder, since there could be other future files that we'd want to include, for the same reason.
Does that sound good to you?

Reply:
Done

Reply:
Do you want to add IDEWorkspaceChecks.plist to the repo?

Reply:
No. I wantn't to do it. But you can do it.

Reply:
I think better that i done less.